test/polygon/asset/polygonAssetERC1155.test.ts

```diff
@@ -137,7 +137,7 @@ describe('PolygonAssetERC1155.sol', function () {
       const tokenId = 42;
       await expect(
         PolygonAssetERC1155.callStatic.uri(tokenId)
-      ).to.be.revertedWith('NFT_!EXIST_||_FT_!MINTED');
+      ).to.be.revertedWith('INVALID_ID');
```

Comment:
changer 'INVALID_ID' en 'Invalid id'

Reply:
no, the revert message is actually INVALID_ID is the contract

Reply:
ok